### DISCUSSION TITLE
Name not added to contributor list after contribution??

### DISCUSSION THREAD

**Discussion by ashutoshgithubs** [ashutoshgithubs/iemhacks_DEV-G#81] [Announcements]
If your pull request (PR) has been successfully merged, but your name or account is not appearing in the contributors list, this could be the reason:

**Your email address is not associated with your GitHub account**: GitHub uses your email address to match your contributions with your GitHub account. If your email address is not associated with your GitHub account, then your contributions may not be recognized. To fix this issue, you can add your email address to your GitHub account by going to your account settings and then selecting "Emails" from the left-hand menu.

**Comment by ashutoshgithubs** [comment]
@maha-sachin can you please address the above issue. Because a few contributors are telling that they have right email connected with gitHub account, but still they are facing this issue. Kindly try to resolve.

**Reply by maha-sachin** [reply]
@ashutoshgithubs , yeah sure I'll check and update here.
